app/models/alaveteli_pro/subscription_with_discount.rb

```diff
@@ -50,7 +42,27 @@ def free?
 
   private
 
+  def coupon?
+    coupon
```

Comment:
I'd expect this to return true or false rather than the object - I think that would be what we'd test for if this were a public method